tracer/src/Datadog.Trace/Telemetry/Metrics/MetricTags.cs

```diff
@@ -230,4 +230,35 @@ public enum WafAnalysis
         [Description("waf_version;rule_triggered:false;request_blocked:false;waf_timeout:true;request_excluded:false")]WafTimeout,
         [Description("waf_version;rule_triggered:false;request_blocked:false;waf_timeout:false;request_excluded:true")]RequestExcludedViaFilter,
     }
+
+    public enum IastInstrumentedSources
+    {
+        [Description("source_type:http.request.body")] RequestBody,
+        [Description("source_type:http.request.path")] RequestPath,
+        [Description("source_type:http.request.parameter.name")] RequestParameterName,
+        [Description("source_type:http.request.parameter")] RequestParameterValue,
+        [Description("source_type:http.request.path.parameter")] RoutedParameterValue,
+        [Description("source_type:http.request.header")] RequestHeader,
+        [Description("source_type:http.request.header.name")] RequestHeaderName,
```

Comment:
It is a header name and value. source_type:http.request.header refers to a header value. It is not very clear, but it is shared with other teams. It is defined here: https://github.com/DataDog/experimental/blob/main/teams/asm/vulnerability_schema/vulnerability_schema.json

Reply:
FWIW, it might be clearer just to update the enum member name (as they can be anything) 🙂 

```suggestion
        [Description("source_type:http.request.header")] RequestHeaderNameAndValue,
        [Description("source_type:http.request.header.name")] RequestHeaderName,
```

Reply:
Sure! Thanks!